sdk/python/test_data/pipelines/if_elif_else_complex.py

```diff
@@ -77,7 +82,7 @@ def lucky_number_pipeline(add_drumroll: bool = True,
                         text='Announcing again: Got the lucky number 5000! A one in 10,000 chance.'
                     )
 
-    print_strings(strings=dsl.Collected(even_or_odd_task.output))
+    print_ints(ints=dsl.Collected(int_task.output))
```

Comment:
The previous state was erroneous and the expected behavior is ambiguous. Updated this example and created a work item to add validation for this case.